src/pages/Home.js

```diff
@@ -0,0 +1,23 @@
+import {
+  checkLocalStorageForKey,
+  addKeyValuePairToLocalStorage,
+} from '../lib/localStorage';
+import { useHistory } from 'react-router-dom';
+
+export default function Home() {
+  const isThereToken = checkLocalStorageForKey('token', '');
```

Comment:
Is this returning a boolean? Naming is hard! If we're receiving a token, we should indicate this. If it's a boolean as `isThereToken` indicates, a more idiomatic name might be `tokenExists` or `tokenPresent`. This is for a few reasons, we're ensuring the subject is the first part or prefix of the name, `token`. The suffix is a descriptor, "what does this boolean indicate? Presence or existence".